src/main/java/com/homework/PrototypeBean.java

```diff
@@ -0,0 +1,26 @@
+package com.homework;
+
+import org.apache.log4j.Logger;
+
+public class PrototypeBean {
+    private static Logger log = Logger.getLogger(PrototypeBean.class.getName());
+    private int id;
+
+    public PrototypeBean() {
+        id = NumberGenerator.getUnigueNumber();
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void initObjectPrototype(){
+        System.out.println("Prototype init..");
```

Comment:
Please don't use System.out.println(). With the help of logger, you could write logs into consol and file.